orchestrator/publisher/application/services/organization_service.py

```diff
@@ -0,0 +1,74 @@
+import json
+
+from common.boto_utils import BotoUtils
+from common.logger import get_logger
+from orchestrator.config import REGION_NAME, REGISTRY_ARN, WALLETS_SERVICE_ARN
+
+logger = get_logger(__name__)
+
+
+class OrganizationOrchestratorService:
+
+    def __init__(self):
+        self.boto_client = BotoUtils(REGION_NAME)
+
+    def register_org_member(self, username, org_uuid, payload):
+        full_name = payload["full_name"]
+        phone_number = payload["phone_number"]
+        wallet_address = payload["wallet_address"]
+
+        self.dapp_update_user_details(full_name, phone_number)
+        self.registry_register_org_member(org_uuid, username, wallet_address)
+        self.register_wallet(username, wallet_address)
```

Comment:
we need to  have a mechanism to handle an exception, what happens if nay f the states fails